app/models/local_delivery_unit.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This is the new (2021) LDU - Local Delivery Unit.
+# The old (incorrectly named) one will be removed in February 2021 after the PDU/LDU go live
+class LocalDeliveryUnit < ApplicationRecord
+  VALID_COUNTRIES = ['England', 'Wales'].freeze
+
+  validates :country, inclusion: { in: VALID_COUNTRIES }
+  validates :code, presence: true, uniqueness: true, format: { with: Team::TEAM_LDU_CODE_REGEX }
```

Comment:
It might be nice to duplicate the validation regex for LDU code format into this model. That way, this new `LocalDeliveryUnit` model will be completely independent of the old LDU/Team models, rather than building in a dependency. Ideally they wouldn't know about each other.